src/StructuredLogViewer.Core/ProjectGraph/RuntimeGraph.cs

```diff
@@ -11,6 +13,7 @@ namespace StructuredLogViewer.Core.ProjectGraph
 {
     public class RuntimeGraph
     {
+        [DebuggerDisplay("{DebuggerDisplayString()}")]

```

Comment:
Can you override ToString()?

It can be used in more situations such as logging, tests, etc. Also DebuggerDisplay makes the debugger eval a tiny bit slower.

Reply:
I'd rather the debugger be slower, than me having to click through various fields to understand what that object is. I'll rename it to ToString.